test/spec_lint.rb

```diff
@@ -604,6 +604,17 @@ def body.to_path; __FILE__ end
     body.to_path.must_equal __FILE__
   end
 
+  it "responds to nil to_path" do
```

Comment:
```suggestion
  it "handles body.to_path returning nil" do
```

Reply:
Thanks for the review.  Updated.

Re the above, I've looked at Ruby's [Precedence](https://msp-greg.github.io/ruby_trunk/file.precedence.html) too often to forget it.  Others may be otherwise, so paren's are a good suggestion...